frontend/src/App.js

```diff
@@ -1,40 +1,22 @@
 import "bootstrap/dist/css/bootstrap.min.css";
-import React, { Component } from "react";
-import { Button, Nav, Navbar } from "react-bootstrap";
+import React from "react";
+import { Route, Switch } from "react-router-dom";
 import "./App.css";
-import Content from "./components/contentMain/content";
-import Footer from "./components/footer/footer";
-import Login from "./components/forms/login";
-import Header from "./components/header/header";
-import Banner from "./components/secBanner/secBanner";
+import PrivateRoute from "./components/PrivateRoute";
+import AllRecipes from "./components/recipePages/allrecipesPage";
+import MainPage from "./Pages";
+import NotFound from "./Pages/404";
+import Dash from "./Pages/Dash";
 
-class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <Navbar className="bg" bg="light" variant="light" sticky="top">
-          <Navbar.Brand href="">WhatToEat</Navbar.Brand>
-          <Navbar.Collapse className="justify-content-end">
-            <Nav>
-              <Button className="button">
-                <Login></Login>
-              </Button>
-            </Nav>
-          </Navbar.Collapse>
-        </Navbar>
-        <Header></Header>
-        <div>
-          <Content></Content>
-        </div>
-        <div>
-          <Banner></Banner>
-        </div>
-        <div>
-          <Footer></Footer>
-        </div>
-      </div>
-    );
-  }
+function App() {
+  return (
+    <Switch>
+      <Route exact path="/" component={MainPage} />
+      <Route exact path="/404" component={NotFound} />
+      <PrivateRoute exact path="/home" component={Dash} />
+      <PrivateRoute exact path="/home/allRecipes" component={AllRecipes} />
```

Comment:
Porque não só /recipes ? O all ja é algo que ja vem por defeito, se for para fazer filtros usas os query params.